### PR TITLE
Add `blogs` folder for blog post code

### PR DESCRIPTION
**Pull request by leahecole**
- Adds new `blog` folder as destination for all Cloud Composer related code found in blog posts
- Adds code for [upcoming blog post in GCP tech blog](https://docs.google.com/document/d/1mvMMs2cmlf0AEwOFviWUUEtWrLOe-XHFWXS48NwRgwQ/edit?resourcekey=0-Paa9Rjr4aOERt7X6EhJzaQ#heading=h.he4owzhp6j8q) (internal only link)

Once I have a link and the post is published, I'll add it to the README in `blog/gcp-tech-blog`

#5358 will resolve some header checker issues, and I'll add the license to the YAML configs now

### CODE REVIEW

**Comment by leahecole** [comment]
@rachael-ds please take a look!

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/requirements.txt
Comment (rachael-ds) [inline composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/requirements.txt:2]: Will the plan be to update these versions manually whenever we have an upgrade?

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag.py
Comment (rachael-ds) [inline composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag.py:46]: Did you want to include the list_buckets_result task in the dependency chain?

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/Dockerfile
Comment (rachael-ds) [inline composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/Dockerfile:20]: I'm not hot on docker files so this may be way off ... The requirements-composer.txt file include everything the requirements.txt file and then some. What's the difference between the two ... as you're coping both I guess we expect them to diverge at some point?

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/renovate.json
Comment (rachael-ds) [inline composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/renovate.json:1]: Just for my own knowledge, what does this file do? :)

**Review by rachael-ds — COMMENTED** [review]
Looks awesome. I'm not an expert on docker so I can't comment much on those files but I've just left a few minor comments elsewhere :)

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/requirements.txt
Comment (rachael-ds) [reply composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/requirements.txt:2]: Already answered in your blog: 
"I specifically pin the operators to a specific version so it is always clear what version is installed in my environment. "

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/Dockerfile
Comment (rachael-ds) [reply composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/Dockerfile:20]: Already answered in your blog:
To keep track of the PyPI modules installed in my Composer environment, I have been using a special `requirements-composer.txt` file that lives in the GitHub repository where I store my dags. Create this file now - it's just like a regular `requirements.txt` file, only with a special name. In it, I've added the most recent veersion of the operator package I'm using - in this case, `apache-airflow-backport-providers-google`

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/requirements.txt
Comment (leahecole) [reply composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/requirements.txt:2]: In general also, we always have a bot watching our versions to upgrade them, so I never have to worry

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/renovate.json
Comment (leahecole) [reply composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/renovate.json:1]: Great question! This is a configuration file for [whitesource renovate bot](https://www.whitesourcesoftware.com/free-developer-tools/renovate/), our robot bff that watches our repos and looks for dependencies that need to be updated

**Comment by leahecole** [comment]
Need to investigate #5392 before I can merge this

**Comment by leahecole** [comment]
test failures have to do with a missing fixture that creates the airflow db. working on it.

**Review by rachael-ds — APPROVED** [review]

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py
Comment (tmatsuo) [inline composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py:27]: Can you get the project id from `GOOGLE_CLOUD_PROJECT` instead of the magic string?

**Review by tmatsuo — CHANGES_REQUESTED** [review]
LGTM with one nit

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py
Comment (leahecole) [reply composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py]: good catch - I realized that I should actually just be using a filler string here - we don't actually need to have our test project and there's no need to grab an environment variable. It's just a dummy variable that needs to exist - it doesn't matter what it's set to.

File: composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py
Comment (tmatsuo) [inline composer/blog/gcp-tech-blog/unit-test-dags-cloud-build/dags/example_dag_test.py:20]: What does this comment mean?

**Review by tmatsuo — APPROVED** [review]

**Comment by leahecole** [comment]
Admin merged because @rachael-ds is a composer codeowner, I'm not sure why GH isn't recognizing as such

**Pull request merged**
